website/addons/base/views.py

```diff
@@ -283,9 +285,41 @@ def create_waterbutler_log(payload, **kwargs):
 
         node_addon.create_waterbutler_log(auth, action, metadata)
 
+    # file indexing
+    metadata = payload.get('metadata') or payload['destination']
```

Comment:
If we're only going to be indexing OsfStorage files, which I believe is the plan, then these hooks should be in `website.addons.osfstorage.model` or `website.addons.osfstorage.views`.